### DISCUSSION TITLE
Image audio simultaneous display?

### DISCUSSION THREAD

**Discussion by rbruno1** [jspsych/jsPsych#1570] [Q&A] [answered]
Is there a way to display image and audio simultaneously?

also, I'm running this in cognition.run, it isn't running but displays no errors. any ideas?
var movie_trial = {
  type:'movie-keyboard-response',
  stimulus: 'Banjo2Bandage1.mp4',  
  choices: ['f', 'j'],
}

var timeline = [movie_trial]

jsPsych.init({
  timeline: timeline
})

**Comment by kurokida** [comment]
About presenting the image and audio stimuli simultaneously, [the psychophysics plugin](https://jspsychophysics.hes.kyushu-u.ac.jp/) might be helpful.
But, the SOA is relatively different from the intended time. Please see [my paper](https://rdcu.be/b5Nie).

**Comment by becky-gilbert** [comment, marked as answer]
Hi @rbruno1, do you have your own custom plugin called "movie-keyboard-response", or do you want to use jsPsych's ["video-keyboard-response" plugin](https://www.jspsych.org/plugins/jspsych-video-keyboard-response/)? If it's the latter, you should change the plugin name in your `movie_trial` object, and use square brackets around the movie file name, because the `stimulus` parameter should be an array.
```js
var movie_trial = {
  type: 'video-keyboard-response',
  stimulus: ['Banjo2Bandage1.mp4'],
  choices: ['f', 'j'],
}

var timeline = [movie_trial]

jsPsych.init({
  timeline: timeline
})
```
